packages/vue/src/components/atoms/SfImage/SfImage.vue

```diff
@@ -138,6 +183,10 @@ export default {
             : `${this.height}px`,
       };
     },
+    imageComponentTag() {
+      if (!this.$nuxt) return "img";      
+      return this.imageTag;
+    }
```

Comment:
semicolon :)

Reply:
suggestion: I think we can simplify with ```return !this.$nuxt ? "img" : this.imageTag```

Reply:
of course, thx